tempesta_fw/http.c

```diff
@@ -615,35 +634,212 @@ tfw_http_adjust_resp(TfwHttpResp *resp, TfwHttpReq *req)
 	return tfw_http_set_hdr_connection(m, conn_flg);
 }
 
+static inline bool
+tfw_http_req_is_idempotent(TfwHttpReq *req)
+{
+	return (req->method != TFW_HTTP_METH_POST);
```

Comment:
According to the new design, the function should return flag for !NON_IDEMPOTENT flag.